CONTRIBUTING.md

```diff
@@ -157,6 +158,60 @@ Include details about your configuration and environment:
     * Problem can be reliably reproduced, doesn't happen randomly: [Yes/No]
     * Problem happens with all files and projects, not only some files or projects: [Yes/No]
 
+### Suggesting Features
```

Comment:
Should we maybe call these "enhancements" to be more consistent with our labeling scheme?